doc/guides/getting_started/src/README.md

```diff
@@ -42,16 +45,16 @@ If you do have Linux, you are still welcome to try the Docker container.
 However, as the container option is currently experimental, we recommend following the steps below to build manually if you plan on being a long-term user or contributor for the project.
 
 Our continuous integration setup runs on Ubuntu 20.04 LTS, which gives us the most confidence that this distribution works out of the box.
-We do our best to support other distributions, but cannot guarantee they can be used "out of the box" and might require updates of packages.
+We do our best to support other distributions, but cannot guarantee they can be used "out of the box" and they might require updates of packages.
 Please file a [GitHub issue](https://github.com/lowRISC/opentitan/issues) if you need help or would like to propose a change to increase compatibility with other distributions.
 
 You will need at least **7GiB of available RAM** in order to build the Verilator simulation.
-If you have an FPGA and download the bitstream from our cloud bucket rather than building it locally (the default setup) then this constraint does not apply.
+If you are building another form of simulation, this constraint does not apply.
 
 If you are specifying a new machine to run top-level simulations of the whole of OpenTitan using Verilator, it is recommended that you
-have a minimum of **32GiB of physical RAM** and at least **512GiB of SSD/HDD storage** for the build tools, repository and Ubuntu installation.
+have a minimum of **32GiB of physical RAM** and at least **512GiB of disk storage** for the build tools, repository and Ubuntu installation.
 
-## Step 2: Install Package Manager Dependencies
+## Step 2: Install dependencies using the package manager
```

Comment:
Low priority but do you think you could go through all the headings and change them to use a consistent capitalization scheme?

Reply:
Hmm, good point. I've done a follow-up with that change here: https://github.com/lowRISC/opentitan/pull/17969.